src/common/hook/usePublishQueue/usePublishQueue.tsx

```diff
@@ -63,14 +63,25 @@ export const usePublishQueue = (
       const deferredJobs = publishingJobs.map(async (job, index) => {
         try {
           const signer = config.wallet;
-          await publishJob(job, signer);
+          if (job.contractAddress === "DNS-DID") {
+            // publish DID verifiable documents first
```

Comment:
can enum the type and share it among the files i guess `DID | DNS-DID | DNS-TXT`